web/src/components/GuidedTour/index.css

```diff
@@ -0,0 +1,3 @@
+.tour-mask {
```

Comment:
Let's move this to a wrapping style component. So we keep the same standard of not using CSS modules.

Reply:
this does not work with styled components